cucumber/authenticators_oidc/features/authn_oidc_bad_policy.feature

```diff
@@ -33,8 +43,9 @@ Feature: OIDC Authenticator - Bad authenticator configuration leads to an error
       role: !group conjur/authn-oidc/keycloak/users
       member: !user alice
     """
-    And I am the super-user
-    And I successfully set provider-uri variable
+    And I set the following conjur variables:
+      | variable_id                             | context_variable  |
+      | conjur/authn-oidc/keycloak/provider-uri | oidc_provider_uri |
     And I fetch an ID Token for username "alice" and password "alice"
     And I save my place in the log file
```

Comment:
@jvanderhoof - with the goal of being more declarative - I'm wondering if this is required as a separate step?  could this be included as part of another step?